scripts/cargo-install-all.sh

```diff
@@ -219,8 +219,8 @@ if [[ -z "$validatorOnly" ]]; then
   "$cargo" $maybeRustVersion build --manifest-path programs/bpf_loader/gen-syscall-list/Cargo.toml
   # shellcheck disable=SC2086 # Don't want to double quote $rust_version
   "$cargo" $maybeRustVersion run --bin gen-headers
-  mkdir -p "$installDir"/bin/sdk/sbf
-  cp -a sdk/sbf/* "$installDir"/bin/sdk/sbf
+  mkdir -p "$installDir"/bin/platform-tools-sdk/sbf
+  cp -a platform-tools-sdk/sbf/* "$installDir"/bin/platform-tools-sdk/sbf
```

Comment:
This is the change I'm least sure about, I'm not sure if it will break any existing tooling to put the sbf sdk in a new directory during installation

Reply:
will we use the sdk dir name for storing sdk related items in the future? If not, I think we can keep this one. if I don't miss anything, this will be located in the end user’s system, so the name might not need to reflect our source 🤔 (line 223)

btw, if we decide to keep the original struct, we may need to update the logic in `cargo-build-sbf` and `cargo-test-sbf`

Reply:
I thought it would be more confusing to use a different directory between the monorepo and the release. My hope is that there's no `sdk/` in agave in the future, and that we can even move `platform-tool-sdk` out too!

I'll keep this then, thanks!